rdkcentral/rdkservices
# RDK-45352 RDK-45346: Upgrade SkyXione-LLAMA & Xumo devices to use Thunder R4.4.1

* RDK-45352 RDK-45346: Upgrade SkyXione-LLAMA & Xumo devices to use Thunder R4.4.1 (#5116)

Reason for change: Compilation Error Fixed of rdkservices plugins against Thunder R4.4.1
Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim Razith tabbas651@cable.comcast.com

  Updated:   DeviceInfo/DeviceInfo.cpp
  Updated:   DeviceInfo/Module.h
  Updated:   Messenger/Module.h
  Updated:   Monitor/Monitor.h
  Updated:   OpenCDMi/FrameworkRPC.cpp
  Updated:   PersistentStore/Module.h
  Updated:   RDKShell/RDKShell.cpp
  Updated:   ResourceManager/ResourceManager.cpp
  Updated:   RustAdapter/LocalPlugin.cpp
  Updated:   RustAdapter/LocalPlugin.h
  Updated:   RustAdapter/RemotePlugin.cpp
  Updated:   RustAdapter/RemotePlugin.h
  Updated:   RustAdapter/RustAdapter.cpp
  Updated:   RustAdapter/RustAdapter.h
  Updated:   SecurityAgent/SecurityAgent.cpp
  Updated:   SecurityAgent/SecurityAgent.h
  Updated:   SystemServices/SystemServices.cpp
  Updated:   SystemServices/platformcaps/platformcapsdata.h
  Updated:   helpers/UtilsJsonRpc.h

(cherry picked from commit e3bf08249bd8c4a4da08b393bd575ba176dfbd16)

RDK-45352 RDK-45346: Update rdkservices plugins to be compatible for Thunder R4.4.1 (#5118)

Reason for change: Updated LocalDispatcher correctly for Invoke method
Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim Razith tabbas651@cable.comcast.com

 Updated:   SystemServices/platformcaps/platformcapsdata.h

* Update platformcapsdata.h

---------

Co-authored-by: Karunakaran A <48997923+karuna2git@users.noreply.github.com>

(cherry picked from commit 87c498d11b61363fd21363100d5f0579c79da4b9)

DELIA-64992: [R4.4.1] [Auto][24Q2]Device struck at Hubbl/Sky/Xumo Logo post CDL (#5142)

Reason for change:  Modify the Invoke method error check with respect to Thunder R4.4.1 code
Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>

XIONE-14377: [ALPACA_DE][UK][Alpaca IT][Foxtel] Ip linear (#5190)

Reason for change: removed the unnecessary dispatcher release from invoke method which landed for R4.4.1
Test Procedure: Verified in Jenkins Build
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>

Co-authored-by: Karunakaran A <48997923+karuna2git@users.noreply.github.com>
(cherry picked from commit 2d3a914313771deb04ea5468571e6f8ac3ac40ce)
